### DISCUSSION TITLE
After restart the minio, the xfsaild state in D

### DISCUSSION THREAD

**Discussion by zhucan** [minio/minio#15663] [Q&A] [answered]
the data dir of minio
![image](https://user-images.githubusercontent.com/13997135/188864862-e897cc14-78f7-491c-a64b-0f30bb5e2f93.png)

xfsaild state
![image](https://user-images.githubusercontent.com/13997135/188864944-3224335f-2efb-451a-b5fc-a14b2ce4c277.png)

iotop print:
![image](https://user-images.githubusercontent.com/13997135/188865138-6912dd09-adc6-475e-8060-be7b679794c6.png)

**Comment by klauspost** [comment, marked as answer]
You provide almost no information. Either way, XFS is just used as a filesystem by MinIO.

If you want us to help debug your OS install, look into our [subscription offer](https://min.io/pricing).
